.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ubuntu-latest, windows-latest, macos-latest]
+        platform: [ubuntu-latest, windows-latest, macos-latest-large]
```

Comment:
Hey @jpal91, I just noticed this is still referencing large and the CI test is failing for that reason I think!

Change it back to `macos-latest` and CI should pass 👍

Reply:
@jacobdeichert Looks like it lines up now. I think I may have merged that file incorrectly or something. CI passes now on my end!